cmd/earthly/org_cmds.go

```diff
@@ -265,3 +302,103 @@ func (app *earthlyApp) actionOrgRevoke(cliCtx *cli.Context) error {
 	}
 	return nil
 }
+
+func (app *earthlyApp) actionOrgMemberList(cliCtx *cli.Context) error {
+	app.commandName = "orgMemberList"
+
+	if cliCtx.NArg() != 0 {
+		return errors.New("expected no arguments")
+	}
+
+	cloudClient, err := cloud.NewClient(app.apiServer, app.sshAuthSock, app.authToken, app.console.Warnf)
+	if err != nil {
+		return errors.Wrap(err, "failed to create cloud client")
+	}
+
+	orgName, err := projectOrgName(cliCtx, cloudClient)
+	if err != nil {
+		return err
+	}
+
+	members, err := cloudClient.ListOrgMembers(cliCtx.Context, orgName)
+	if err != nil {
+		return err
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+	for _, member := range members {
+		fmt.Fprintf(w, "%s\t%s\n", member.UserEmail, member.Permission)
+	}
+	w.Flush()
+
+	return nil
+}
+
+func (app *earthlyApp) actionOrgMemberUpdate(cliCtx *cli.Context) error {
+	app.commandName = "orgMemberUpdate"
+
+	if cliCtx.NArg() != 2 {
+		return errors.New("member email and new permission required")
+	}
+
+	cloudClient, err := cloud.NewClient(app.apiServer, app.sshAuthSock, app.authToken, app.console.Warnf)
+	if err != nil {
+		return errors.Wrap(err, "failed to create cloud client")
+	}
+
+	orgName, err := projectOrgName(cliCtx, cloudClient)
+	if err != nil {
+		return err
+	}
+
+	userEmail := cliCtx.Args().Get(0)
+	if userEmail == "" {
+		return errors.New("member email required")
+	}
+
+	permission := cliCtx.String("permission")
```

Comment:
Mind using a variable here like we do for everything else?

Reply:
Sure. That big arg struct is bothering me a bit :P

Reply:
Yeah... we should categorize flags a bit better soon...